test/Makefile

```
@@ -26,7 +26,7 @@ test-e2e: run ## Runs the E2E tests
 
 .PHONY: test-unit
 test-unit: ## Runs the unit tests, skips e2e
-	go test `go list ../... | grep -v /e2e`
+	go test -coverprofile coverage.out `go list ../... | grep -v /e2e`
```

E2E tests and framework are captured within `/test` folder, so I'd suggest skipping that one:

```suggestion
	go test -coverprofile coverage.out `go list ../... | grep -v /test`
```

---

But the rood dir for the grep command is "test", not the repo root

---

But still, we are changing context with `../...` in `go list`, because we are executing the target from `test/Makefile`. Right?